ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerTest.scala

```diff
@@ -45,11 +45,21 @@ object BigSamplerTest extends Properties("BigSampler") {
     val dice2 = BigSampler.diceElement(i, hasher2.hash(), 1.0)
     dice1 == dice2
   }
+`
+  private val negativeHashCodeStringGen: Gen[String] = Gen.alphaStr
+    .suchThat(str => { newTestHasher().putString(str, BigSampler.utf8Charset).hash().asLong < 0} )
+  property("dice on hash values that return negative Long representations should still bucket OK") =
```

Comment:
This is awesome, can we have a more specific title? Or just a better word that `OK`